src/commands/lambda/function.ts

```diff
@@ -244,6 +233,28 @@ export const calculateUpdateRequest = (
   return needsUpdate ? updateRequest : undefined
 }
 
+const determineIfLayerShouldBeAdded = (
```

Comment:
The name is kind of confusing given the return type. I would expect it to return a boolean based on that name. Maybe just addLayerARN? I think you could also keep the extension/env var check in calculateUpdateRequest, since it's not related to what the function indicates it's doing.

Reply:
yeah I was a bit iffy on the name, thanks for pointing it out. I also moved the extension stuff back into calculateUpdateRequest